torch/onnx/symbolic_opset11.py

```diff
@@ -791,7 +791,8 @@ def narrow(g, input, dim, start, length):
     end = g.op("Add", start, length)
     return _slice_helper(g, input, axes=dim, starts=start, ends=end, dynamic_slice=True)
 
-
+from torch.onnx.symbolic_helper import quantized_args
```

Comment:
please move this import to line 10. It's quite plausible some other symbolic will need it at some point.